common/djangoapps/student/tasks.py

```diff
@@ -6,12 +6,46 @@
 from boto.exception import NoAuthHandlerFound
 from celery.exceptions import MaxRetriesExceededError
 from celery.task import task  # pylint: disable=no-name-in-module, import-error
+from django.apps import apps
 from django.conf import settings
 from django.core import mail
+from django.contrib.auth.models import User
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 
 log = logging.getLogger('edx.celery.task')
 
 
+@task()
+def send_payment_link_to_parent(user_id):
+    """
+    Sending an payment link email to the parent.
+    """
+    ParentProfile = apps.get_model('tedix_ro', 'ParentProfile')
+    try:
+        user = User.objects.get(pk=user_id)
+        parent = user.parentprofile
+    except ParentProfile.DoesNotExist:
+        log.error("send_payment_link_to_parent: This user has no parentprofile.")
+    except User.DoesNotExist:
+        log.error("send_payment_link_to_parent: User with this ID does not exist.")
+    
+    from_address = configuration_helpers.get_value(
+        'email_from_address',
+        settings.DEFAULT_FROM_EMAIL
+    )
+    if user.is_active and parent:
```

Comment:
`parent` and `user` won't be available in case the exception occurs, you should move all this logic under try block